<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-LIAKTO4M" failures="1" tests="8" name="Validations" time="111.867" errors="0" timestamp="22 Nov 2021 09:00:09 GMT">
  <testcase classname="testRunner.TestRunner" name="runScenario" time="13.163"/>
  <testcase classname="testRunner.TestRunner" name="runScenario" time="23.739"/>
  <testcase classname="testRunner.TestRunner" name="runScenario" time="17.0"/>
  <testcase classname="testRunner.TestRunner" name="runScenario" time="7.603">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
	at com.mindtree.pageObject.HomePage.validateLogin(HomePage.java:51)
	at stepDefinition.LoginStepDefinition.verify_that_the_user_is_unable_to_log_in_to_the_page(LoginStepDefinition.java:28)
	at âœ½.verify that the user is unable to log in to the page(file:///C:/Users/Spoorthy/eclipse-workspace/BigSmallAppBDD/src/test/java/feature/Login.feature:9)
... Removed 4 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="testRunner.TestRunner" name="runScenario" time="14.447"/>
  <testcase classname="testRunner.TestRunner" name="runScenario" time="14.575"/>
  <testcase classname="testRunner.TestRunner" name="runScenario" time="11.481"/>
  <testcase classname="testRunner.TestRunner" name="runScenario" time="9.274"/>
</testsuite> <!-- Validations -->
